pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/TemplateDao.java

```diff
@@ -171,53 +179,43 @@ default void disableTemplate(long templateId, RevisionMetadata meta) {
     Map<Long, TextAndVarCount> findAllTextAndVarCountsByIds(@BindList("templateIds") Set<Long> templateIds);
 
     default Template loadTemplateById(long templateId) {
-        String query = StringTemplateSqlLocator
-                .findStringTemplate(TemplateDao.class, "queryTemplateById")
-                .render();
+        return loadTemplatesByIds(Set.of(templateId)).findFirst().orElse(null);
```

Comment:
Any call sites that uses the single template query will exercise the bulk query instead. The bulk query will be used more extensively in later PRs.